web/tests/home.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { test, expect } from '@playwright/test'
+
+test.beforeEach(async ({ page }) => {
+  await page.goto('/')
+})
+
+test('has title', async ({ page }) => {
+  await expect(page).toHaveTitle(/CoVariants/)
+})
+
+test('navbar link', async ({ page }) => {
+  await page.getByRole('link', { name: 'FAQ' }).click()
+
+  await expect(page.getByRole('heading', { name: 'Frequently asked questions' })).toBeVisible()
+})
```

Comment:
I usually like to group those into describe block, because it yields nicer descriptions when running the tests and the code is grouped logically.
```suggestion
test.describe('The Home page', () => {
test.beforeEach(async ({ page }) => {
  await page.goto('/')
})

test('has title', async ({ page }) => {
  await expect(page).toHaveTitle(/CoVariants/)
})

test('navbar link', async ({ page }) => {
  await page.getByRole('link', { name: 'FAQ' }).click()

  await expect(page.getByRole('heading', { name: 'Frequently asked questions' })).toBeVisible()
})
});
```